libs/modules/sessions.js

```diff
@@ -0,0 +1,136 @@
+const path = require('path');
+const common = require('../common');
+const Storage = require('../jsonstore');
+const JSONStore = Storage.JSONStore;
+const Storable = Storage.Storable;
+const expect = common.expect;
+
+class Session extends Storable {
```

Comment:
do we want to care about sessions? Git is stateless so we might want to follow the same principles.

Reply:
Just because it exists in the fw doesn't mean it's required to use it. 
Just set sessions: false on App.create. :wink:

Reply:
Alternatively just remove the sessions config block from the app config entirely and they won't be used.